tests/test_bs.py

```diff
@@ -455,3 +410,75 @@ def test_transaction(setup_clients):
     print("test took {:.3f} seconds ".format(end-start))
     assert '{"value":"100"}' in out.decode("utf-8")
     print("balance ok")
+
+
+def test_mining(setup_clients, wait_genesis):
+    timetowait = len(setup_clients.pods)/2
+    print("Sleeping " + str(timetowait) + " before checking out bootstrap results")
+    time.sleep(timetowait)
+
+
+    # choose client to run on
+    client_ip = setup_clients.pods[0]['pod_ip']
+
+    api = 'v1/nonce'
+    data = '{"address":"1"}'
+    print("checking nonce")
+    out = api_call(client_ip, data, api, testconfig['namespace'])
+    # assert '{"value":"0"}' in out.decode("utf-8")
+    # print("nonce ok")
+
+    api = 'v1/submittransaction'
+    data = '{"srcAddress":"1","dstAddress":"222","nonce":"0","amount":"100"}'
+    print("submitting transaction")
+    out = api_call(client_ip, data, api, testconfig['namespace'])
+    print(out.decode("utf-8"))
+    assert '{"value":"ok"}' in out.decode("utf-8")
+    print("submit transaction ok")
+    print("wait for confirmation ")
+    api = 'v1/balance'
+    data = '{"address":"222"}'
+    end = start = time.time()
+    layer_avg_size = 20
+    last_layer = 8
+    layers_per_epoch = 3
+    deviation = 0.2
+    last_epoch = last_layer / layers_per_epoch
+
+    queries.wait_for_latest_layer(testconfig["namespace"], last_layer)
+
+    # out = api_call(client_ip, data, api, testconfig['namespace'])
+    print("test took {:.3f} seconds ".format(end-start))
+    # assert '{"value":"100"}' in out.decode("utf-8")
+    # print("balance ok")
+
+    # need to filter out blocks that have come from last layer
+    blockmap = queries.get_blocks_per_node(testconfig["namespace"])
+    # count all blocks arrived in relevant layers
+    total_blocks = sum([len(blockmap[x]) for x in blockmap ])
+    atxmap = queries.get_atx_per_node(testconfig["namespace"])
+    total_atxs = sum([len(atxmap[x]) for x in atxmap ])
+
+    total_pods = len(setup_clients.pods) + 1
```

Comment:
I would rather that we'll actually count the bootstrap node instead of adding 1. at least add a comment about it.